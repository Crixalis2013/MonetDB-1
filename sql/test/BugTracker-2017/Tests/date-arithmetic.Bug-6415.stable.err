stderr of test 'date-arithmetic.Bug-6415` in directory 'sql/test/BugTracker-2017` itself:


# 17:25:33 >  
# 17:25:33 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-441011" "--port=36103"
# 17:25:33 >  

MAPI  = (monetdb) /var/tmp/mtest-441254/.s.monetdb.36643
QUERY = select '1' + '1';
ERROR = !SELECT: no such binary operator 'sys'.'sql_add'(char,char)
CODE  = 42000

# 17:25:33 >  
# 17:25:33 >  "Done."
# 17:25:33 >  

