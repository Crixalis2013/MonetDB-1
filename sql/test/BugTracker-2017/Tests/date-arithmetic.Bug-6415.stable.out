stdout of test 'date-arithmetic.Bug-6415` in directory 'sql/test/BugTracker-2017` itself:


# 17:25:33 >  
# 17:25:33 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-441011" "--port=36103"
# 17:25:33 >  

#select cast('2017-10-10' as date) - cast('2017-10-01' as date), date '2017-10-10' - date '2017-10-01';
% .%2,	.%3 # table_name
% %2,	%3 # name
% day_interval,	day_interval # type
% 10,	10 # length
[ 777600.000,	777600.000	]
#select cast('2017-10-01' as date) + interval '9' day, date '2017-10-01' + interval '9' day;
% .%2,	.%3 # table_name
% %2,	%3 # name
% date,	date # type
% 10,	10 # length
[ 2017-10-10,	2017-10-10	]
#select cast('2017-10-01' as date) + interval '777600' SECOND, date '2017-10-01' + interval '777600' SECOND;
% .%2,	.%3 # table_name
% %2,	%3 # name
% date,	date # type
% 10,	10 # length
[ 2017-10-10,	2017-10-10	]

# 17:25:33 >  
# 17:25:33 >  "Done."
# 17:25:33 >  

